### ISSUE TITLE
Logo not visible in 21.0.1 if an accessibility theme is used

### ISSUE THREAD

**Issue by zeidspex** [nextcloud/nextcloud-snap#1743]
#### Describe the bug
The Nextcloud logo does not show up in the top banner if an accessibility theme is turned on. Any accessibility theme triggers this issue, including the Dark theme, High-contrast mode, or Dyslexia font.

#### To Reproduce
1. Install Nextcloud 21.0.1 using snaps.
2. Goto user settings
3. Select accessibility
4. Enable an accessibility theme (ie. Dark theme, High-contrast mode, or Dyslexia font)
5. Refresh the page
6. Observe that the Nextcloud logo will disappear from the top banner

#### Expected behavior
The Nextcloud log should remain visible in the top banner

#### Screenshots
[See screenshot](https://ibb.co/KDQxxpZ)

#### OS/snapd/snap version
I am using Ubuntu 20.04.02 LTS, Nextcloud 21.0.1snap1 Rev. 27920 from the latest/stable channel. I am using snap version 2.50.

**Comment by adrianvg** [comment]
I don't see that. 
Running the same Nextcloud version on Ubuntu Server 20.04.

![2021-05-26_12-50](https://user-images.githubusercontent.com/15629475/119648025-4245c900-be21-11eb-8798-5ff0babdd9af.png)

**Comment by vawaver** [comment]
I can confirm the problem reported above.
When I enable the dark theme -> Logo disappear.

**Comment by zeidspex** [comment]
@adrianvg Is that using the snap? I believe the problem only occurs if you use the snap version.

**Comment by adrianvg** [comment]
> @adrianvg Is that using the snap? I believe the problem only occurs if you use the snap version.

Yup, using the snap.
I've refreshed, logged on and off a few times and also emptied the browser cache.
The logo is still there.

For further checking on my part; except for enabling the dark theme in Accessibility do I have to enable or disable something else as well?

**Comment by zeidspex** [comment]
I tested this bug on a brand new Ubuntu 20.04 LTS virtual machine, and this behavior happened with everything on defaults (including installing the default recommended apps). I also tried to uninstall, then reinstall the snap, and choosing to NOT install the recommended apps. The bug still happened.

It seems on my end, the bug is reproduced reliably, and does not depend on other applications being installed or not.

**Comment by kyrofa** [comment]
Confirmed, I see this as well when enabling the dark theme. That appears to be a bug with the dark theme though, not the snap. I suggest logging this [upstream](https://github.com/nextcloud/server).

**Comment by zeidspex** [comment]
@kyrofa I am not sure if it a bug with the theme or the snaps. However, it seems to be only triggered when using the snaps. I have tried to reproduce this bug in other types of installations, and I have not observed that bug occurring in either docker or manual installations.

**Comment by kyrofa** [comment]
That's fair, it might have something to do with the theme making some assumption about where its assets are. It's a pretty classic bug triggered by the snap, but not one we cause, and not one we can fix here.
